session-6/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { Fragment, useEffect, useState, useRef } from "react";
+import ReactDOM from "react-dom/client";
+import Card from "./components/Card/Card";
+import Header from "./components/Header/Header";
+import membersGitHubUserNames from "./data/members.json";
+import "./App.css";
```

Comment:
You can group your imports by separating them with new lines and adding comments

```suggestion
import React, { Fragment, useEffect, useState, useRef } from "react";
import ReactDOM from "react-dom/client";

import Card from "./components/Card/Card";
import Header from "./components/Header/Header";

import membersGitHubUserNames from "./data/members.json";
import "./App.css";
```